pkg/data/egress_lists/egress_lists.go

```diff
@@ -58,10 +67,12 @@ func GetGithubEgressList(platformType cloud.Platform) (*github.RepositoryContent
 		path += cloud.AWSHCP.String()
 	case cloud.AWSClassic:
 		path += cloud.AWSClassic.String()
+	case cloud.AWSHCPZeroEgress:
+		path += cloud.AWSHCPZeroEgress.String()
 	default:
 		return nil, fmt.Errorf("no egress list registered for platform '%s'", platformType)
 	}
-	fileContentResponse, _, _, err := ghClient.Repositories.GetContents(context.TODO(), "openshift", "osd-network-verifier", fmt.Sprintf("%s.yaml", path), nil)
+	fileContentResponse, _, _, err := ghClient.Repositories.GetContents(context.TODO(), "bmeng", "osd-network-verifier", fmt.Sprintf("%s.yaml", path), nil)
```

Comment:
is this what you were using during testing?

Reply:
ah, yes, this is for testing... I thought I should have updated back, not sure if it was missing during rebase